doc/classes/LineEdit.xml

```diff
@@ -365,7 +365,7 @@
 		<signal name="text_submitted">
 			<param index="0" name="new_text" type="String" />
 			<description>
-				Emitted when the user presses [constant KEY_ENTER] on the [LineEdit].
```

Comment:
This change is unrelated *technically*

If we want to improve upon this description  , this is still too broad and should probably mention `ui_text_submit`, but again, if I want to be nitpicky, it's unrelated.

Reply:
I've split this change to its own PR: https://github.com/godotengine/godot/pull/97455